src/main/java/com/paskar/email/application/service/EmailRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,67 @@
+package com.paskar.email.application.service;
+
+import com.paskar.email.application.mapper.EmailMapper;
+import com.paskar.email.application.model.Email;
+import com.paskar.email.application.repositiory.EmailRepositoryJdbc;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class EmailRepositoryJdbcImpl implements EmailRepositoryJdbc {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public EmailRepositoryJdbcImpl(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public void save(Email email) {
+        String sql = "INSERT INTO first_homework_with_jdbc (recipient,subject, body, date) VALUES (?,?,?,?)";
+        jdbcTemplate.update(sql, email.getRecipient(), email.getSubject(), email.getBody(), email.getDate());
+    }
+
+    @Override
+    public Email getById(int id) {
+        String sql ="SELECT * FROM first_homework_with_jdbc WHERE id=?";
```

Comment:
1) Avoid * symbol.
Your model can represent not whole table, but just some piece of it.
So always specify exactly what you need to retrieve.

2) No need to write `id=?`, write with formatting `id = ?`